.devcontainer/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:24.04
+
+RUN apt update -y && apt install -y \
+	gcc \
+	g++ \
+	cmake \
+	ninja-build \
+	git \
+	libopenblas-dev \
+	&& apt clean && rm -rf /var/lib/apt/lists/*
+
+# Install SuiteSparse for klu
+RUN git clone --single-branch --depth 1 --branch v7.10.2 https://github.com/DrTimothyAldenDavis/SuiteSparse.git /tmp/suitesparse && \
+	cd /tmp/suitesparse && \
+	mkdir -p build && cd build && \
+	cmake -G Ninja -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+		-DSUITESPARSE_ENABLE_PROJECTS="klu" .. && \
+	ninja && \
+	ninja install && \
+	rm -rf /tmp/suitesparse
+
+# Install sundials using klu from previous step
+RUN git clone --single-branch --branch develop https://github.com/LLNL/sundials.git /tmp/sundials && \
+	cd /tmp/sundials && \
+	git checkout 07d21c2c5ae33211a9a2fafd8eac56de5582dce0 && \
+	mkdir -p build && cd build && \
+	cmake -G Ninja -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+		-DENABLE_KLU=ON \
+		-DKLU_ROOT=/usr/local/lib/cmake/KLU .. && \

```

Comment:
`KLU_ROOT` isn't necessary here. cmake is capable of detecting it.

Reply:
Interesting. Did you test this? It wasn’t the case when I’ve installed on my machine and others’.

Reply:
it worked fine in the container. i can test it again but it worked with the ubuntu base image as well as the alpine base image